docs/resources/Guild.md

```diff
@@ -831,12 +831,12 @@ Modify attributes of a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object)
-| channel_id                   | snowflake           | id of channel to move user to (if they are connected to voice)                                                                                                                                                                           | MOVE_MEMBERS     |
-| communication_disabled_until | ?ISO8601 timestamp  | when the user's [timeout](https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ) will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout | MODERATE_MEMBERS |
+| nick                         | string              | value to set user's nickname to                                                                                                                                                                                                                                                                                       | MANAGE_NICKNAMES |
+| roles                        | array of snowflakes | array of role ids the member is assigned                                                                                                                                                                                                                                                                             | MANAGE_ROLES     |
+| mute                         | boolean             | whether the user is muted in voice channels. Will throw a 400 error if the user is not in a voice channel                                                                                                                                                                                                             | MUTE_MEMBERS     |
+| deaf                         | boolean             | whether the user is deafened in voice channels. Will throw a 400 error if the user is not in a voice channel                                                                                                                                                                                                         | DEAFEN_MEMBERS   |
+| channel_id                   | snowflake           | id of channel to move user to (if they are connected to voice)                                                                                                                                                                                                                                                       | MOVE_MEMBERS     |
+| communication_disabled_until | ?ISO8601 timestamp  | when the user's [timeout](https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ) will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout. Will throw a 403 error if the user has administrator permissions | MODERATE_MEMBERS |
```

Comment:
```suggestion
| communication_disabled_until | ?ISO8601 timestamp  | when the user's [timeout](https://support.discord.com/hc/en-us/articles/4413305239191-Time-Out-FAQ) will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout. Will throw a 403 error if the user has `ADMINISTRATOR` permission | MODERATE_MEMBERS |
```

Reply:
Server owners **always** have `Administrator` (`ADMINISTRATOR`) permission.
I don't see the difference between "`ADMINISTRATOR` permission" and your so-called "administrator permissions", except the latter is not consistent with rest of documentation which describes permissions `LIKE_THIS`, and even if there is another difference, it's very hard to see it since it's not intuitive, and [no other part of documentation has used the latter](https://github.com/discord/discord-api-docs/search?q=administrator+permissions), except the one referring to Windows' administrator permissions.
I also would like to recommend you seeing this before you throw same or similar exception to me again:
![image](https://user-images.githubusercontent.com/43488966/151685527-add8f316-4b1b-4c8a-82d9-144c01f27683.png)

Reply:
> "if the user has the `ADMINISTRATOR` permission or is the owner of the guild"

👍 went with this, should be good to merge now